gradle/config/spotbugs/exclude.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!-- http://spotbugs.readthedocs.io/en/latest/filter.html -->
+<FindBugsFilter>
+
+    <Match>
+        <!-- spotbugs thinks email/spendEnabled are "Unwritten public or
+         protected field". They are written when JacksonMapper reads from JSON
+         file. -->
```

Comment:
We've typically used annotations directly in the class/method to suppress false positives for static analysis ([example use](https://github.com/DataBiosphere/terra-cli/blob/6fc7eb78408fbabedd8eac4d8e63c13cdd3c2765/src/main/java/bio/terra/cli/serialization/userfacing/input/GcsBucketLifecycle.java#L20)). Perhaps something to check with the team about whether there's a norm/convention to follow here.